compiler/luci/pass/src/PropagateQuantParamPass.cpp

```diff
@@ -91,9 +91,8 @@ bool PropagateQuantParamPass::run(loco::Graph *g)
     INFO(l) << "PropagateQuantParamPass visit node: " << circle_node->name() << std::endl;
 
     PropagateQuantParam pqp;
-    changed = circle_node->accept(&pqp);
-    if (changed)
-      break;
```

Comment:
@jinevening , did you have any problem with this Pass to use break here?

Reply:
No problem.